pkgs/top-level/all-packages.nix

```diff
@@ -290,6 +290,8 @@ in
 
   fetchMavenArtifact = callPackage ../build-support/fetchmavenartifact { };
 
+  find-libs = callPackage ../development/tools/misc/find-libs { };
```

Comment:
What is the use case for this? Note there are other tools like ldd-tree if you just want to use it as a tool.

Reply:
I built this tool using it: https://gist.github.com/ryantrinkle/ad2cea1435b813ebd3498a950bc11f9a

I'd be very happy to know if there's a better way to do that, though!

Reply:
fair enough